src/initial-submission/hooks/useSupportingFileHook.ts

```diff
@@ -10,6 +10,7 @@ const hook = (
     initialValues: Submission,
     maxSupportingFiles: number,
     maxFileSize: number,
+    uploadProgressData: any,
```

Comment:
Should this be typed to something other than any?

Reply:
It's not typed in FileDetailsForm. So we either type it in both or leave as is

Reply:
Probably best to type it in both then.

Reply:
Types added.